backend-diagram-converter/core/src/main/resources/message-templates.properties

```diff
@@ -50,6 +51,12 @@ completion-condition.severity=REVIEW
 condition-expression.message=Condition expression: {{ templates.expression-transformation-result }}
 condition-expression.severity=REVIEW
 #
+condition-expression-execution.message=Condition expression: {{ templates.expression-transformation-hint }} 'execution' is not available in FEEL.
+condition-expression-execution.severity=TASK
+#
+condition-expression-method.message=Condition expression: {{ templates.expression-transformation-hint }} Method invocation is not possible in FEEL.
+condition-expression-method.severity=TASK
+#
```

Comment:
Did you also include the `completion-condition` expression?

Reply:
Will do. Thank you for the heads-up.

Reply:
Done